flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/TwoInputSelectionHandler.java

```diff
@@ -70,6 +70,10 @@ boolean areAllInputsSelected() {
 		return inputSelection.areAllInputsSelected();
 	}
 
+	boolean isInputUnavailable(int inputIndex) {
```

Comment:
nit: I would flip the method to `isInputAvailable(...)` and use it with `!` operator

Reply:
I started with what you said, but then it is only ever used negated, that's why I flipped. If you prefer `isInputAvailable` I can do change that.

Reply:
+0.1 for `isInputAvailable` from my side. `isInputUnavailable` sounds a bit strange :)

Reply:
I removed the commit, so I am not adding any of the two methods ;)